docs/systemd/polylith-example@.service

```
@@ -0,0 +1,14 @@
+[Unit]
+	Description= Dendrite Polylith Multi - %I
+	PartOf=polyDendrite.service
```

I'm not a fan of camelCase in unit files. Maybe change that to `PartOf=dendrite-polylith.service` (do we even need that?)
```suggestion
	PartOf=dendrite-polylith.service
```